recipes/rte-rrtmgp/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "rte-rrtmgp" %}
+{% set version = "v1.9.0" %}
+{% set run_validation_plots = false %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/earth-system-radiation/rte-rrtmgp/archive/refs/tags/v1.9.zip
+  sha256: 7d61d67b562958b4b6824196ac27673e0ecde297878cb948de647d6e4f70bcc8
+
+build:
+  number: 0
+  script_env:
+    - RUN_VALIDATION_PLOTS={{ run_validation_plots }}
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - python
+    - netcdf-fortran >=4.5,<5.0
+    - netcdf4
+    - xarray
+    - dask
+    - numpy
+    - matplotlib          # [run_validation_plots]
+    - colorcet            # [run_validation_plots]
+    - scipy               # [run_validation_plots]
+    - gcc_linux-64        # [linux]
+    - gxx_linux-64        # [linux]
+    - gfortran_linux-64   # [linux]
+    - clang_osx-64        # [osx and x86_64]
+    - clangxx_osx-64      # [osx and x86_64]
+    - gfortran_osx-64     # [osx and x86_64]
+    - clang_osx-arm64     # [osx and arm64]
+    - clangxx_osx-arm64   # [osx and arm64]
+    - gfortran_osx-arm64  # [osx and arm64]
+    - gcc_win-64          # [win]
+    - gxx_win-64          # [win]
+    - gfortran_win-64     # [win]
+  host:
+    - cmake
+    - python
+    - netcdf-fortran >=4.5,<5.0
```

Is this being linked to?

If so, would drop the constraint

We handle this through [a global pin]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/af79d726dd3e82521880f3e24725a6ba6156b6f7/recipe/conda_build_config.yaml#L723-L724 ) and do periodic migrations to update as needed

```suggestion
    - netcdf-fortran
```